_episodes/03-monitor-the-model.md

```diff
@@ -568,15 +493,13 @@ def create_nn(n_features, n_predictions):
     # Dense layers
     layers_dense = keras.layers.BatchNormalization()(layers_input)
     layers_dense = keras.layers.Dense(100, 'relu')(layers_dense)
-    layers_dense = keras.layers.Dropout(rate=0.2)(layers_dense)
     layers_dense = keras.layers.Dense(50, 'relu')(layers_dense)
-    layers_dense = keras.layers.Dropout(rate=0.2)(layers_dense)
 
     # Output layer
     layers_output = keras.layers.Dense(n_predictions)(layers_dense)
 
     # Defining the model and compiling it
-    return keras.Model(inputs=layers_input, outputs=layers_output, name="model_dropout_batchnorm")
+    return keras.Model(inputs=layers_input, outputs=layers_output, name="model_batchnorm")
 
 model = create_nn(X_data.shape[1], 1)
 model.compile(loss='mse', optimizer=Adam(1e-4), metrics=[keras.metrics.RootMeanSquaredError()])
```

Comment:
Do we need to specify learning rate here? In the previous code snippets we did `optimizer='adam'`, and the learning rate is not yet explained.

Reply:
True, that's historical I guess.... I set it to the simple default `adam`